### PR TITLE
make it possible to pass slim options

### PR DESCRIPTION
**Pull request by pawelniewie**
I found this useful and love to merge it upstream.

### CODE REVIEW

**Comment by MaxPleaner** [comment]
Thanks for doing this @pawelniewie I will try and test this out sometime over the next couple days. Feel free to add sometime to the README about the new feature as well.

File: README.md [hunk 1]
```diff
@@ -38,3 +38,20 @@ that expects javascript or coffeescript input_):
      var file = require("html-loader!./test.slim");
      alert(file); // this will be a html string that auto-reloads 
     ```
+
+## slim options
```
Comment (pawelniewie) [inline README.md]: @MaxPleaner I already did here, anything else you think would be worth adding?

**Comment by MaxPleaner** [comment]
I'm just gonna give you the benefit of the doubt here. You're welcome to use the master branch off github here, will let you know when I push to NPM. Sorry for the delay, I've been preoccupied with other things.

**Comment by MaxPleaner** [comment]
@pawelniewie  this is deployed to NPM with version 0.0.9

**Pull request merged**